### PR TITLE
wmbattery: add 2.54.

### PR DESCRIPTION
**Pull request by a3a3el**

### CODE REVIEW

File: _posts/1970-01-01-wmbattery.md [hunk 1]
```diff
@@ -26,14 +26,17 @@ versions:
   number: 2.49
   download: wmbattery-2.49.tar.gz
  -
-  number: "2.50"
+  number: 2.50
```
Comment (d-torrance) [inline _posts/1970-01-01-wmbattery.md]: These quotes make sure that Jekyll prints the trailing zero, i.e., 2.50 instead of 2.5.  Let's keep them.

**Comment by a3a3el** [comment]
Ah, right, I did wonder.  Will fix.

File: _posts/1970-01-01-wmbattery.md (hunk 1 above)
Comment (a3a3el) [reply _posts/1970-01-01-wmbattery.md]: Ah, right, I did wonder. Will fix.

Comment (a3a3el) [reply _posts/1970-01-01-wmbattery.md]: Amended and force-pushed.

**Comment by d-torrance** [comment]
Thanks!

**Pull request merged**